com.unity.ml-agents/Runtime/Communicator/RpcCommunicator.cs

```diff
@@ -13,10 +16,11 @@
 using Unity.MLAgents.CommunicatorObjects;
 using Unity.MLAgents.Sensors;
 using Unity.MLAgents.SideChannels;
-using Google.Protobuf;
 
 namespace Unity.MLAgents
 {
+#if UNITY_EDITOR || UNITY_STANDALONE_WIN || UNITY_STANDALONE_OSX || UNITY_STANDALONE_LINUX
```

Comment:
This moves the "inline" platform checks, and just defines a stub implementation on non-desktop platforms (below)